plugins/NebulaTextures/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# This is the cmake config file for the HelloStelModule
```

Comment:
The placeholder module names should be replaced with the actual name.

Reply:
Thanks.